rule.php

```diff
@@ -96,8 +96,8 @@ public function make_modal_content() {
     public function add_preflight_check_form_fields(mod_quiz_preflight_check_form $quizform, MoodleQuickForm $mform, $attemptid) {
         global $PAGE;
         $coursedata = $this->get_courseid_cmid_from_preflight_form();
-        $screenshotdelay = get_invigilator_settings('screenshotdelay');
-        $screenshotwidth = get_invigilator_settings('screenshotwidth');
+        $screenshotdelay = quizaccess_invigilator_get_invigilator_settings('screenshotdelay');
```

Comment:
@shadman-ahmed-bs23  can we use get config method directly without introduce another function get_invigilator_settings

Ex. get_config('quizaccess_invigilator', 'screenshotdelay')

Reply:
@sahedbs23 bhai, replaced the function with get_config(). Please check